apps/web/src/lang/fr.json

```diff
@@ -3135,6 +3143,10 @@
     "defaultMessage": "Détails sur l'offre d'emploi",
     "description": "Form header for filling in job opportunity information section."
   },
+  "FNvBxZ": {
+    "defaultMessage": "Voici l’information qui a été soumise le {submittedAt}. Vous trouverez l’historique de la carrière du candidat/de la candidate dans la section sur l’échéancier.",
```

Comment:
The translation of the word **timeline** (_l’échéancier_) is problematic because **career timeline** has been translated as _parcours professionnel_ and so the word _échéancier_ will not actually appear in French, which this copy makes reference to that section.